fab/Actor/Map/AwareTrait.php

```diff
@@ -12,7 +12,7 @@ trait AwareTrait
     public function setActorMap(MapInterface $Actors): self
     {
         if ($this->hasActorMap()) {
-            throw new \LogicException('Actors is already set.');
+            throw new \LogicException('ActorMap is already set.');
```

Comment:
Thoughts on updating these all (and our templates, best practices, etc) to use `__CLASS__` or `__TRAIT__`?
e.g. https://3v4l.org/lHiNg

Reply:
e.g. this?
```suggestion
            throw new \LogicException(__CLASS__ . ' ActorMap is already set.');
```

Reply:
There's some subtleties to the behavior of those: https://www.php.net/manual/en/language.constants.magic.php

Reply:
(updated suggestion in comment to include both)

Reply:
Was thinking more along the lines of

```suggestion
            throw new \LogicException(__TRAIT__ . ' is already set.');
```

or

```suggestion
            throw new \LogicException(__CLASS__ . ' is already set.');
```

so that we don't have to remember to update the string of the thing that "is already set" every time, reducing copy-paste errors.

Reply:
definitely a non-blocker, just exploring the idea.

Reply:
I guess I wasn't really thinking through the semantics of these Aware Traits and what they're doing.
I think `__TRAIT__` could still make sense for these AwareTrait types, though maybe it's a stretch.
You could get fancy by trimming `__FUNCTION__` a bit, but that's getting too clever for my tastes.

Reply:
> reducing copy-paste errors.

These are all fabbed files.

Reply:
:zoop: